firefox-ios/Client/Frontend/Browser/Tabs/Views/RemoteTabsTableViewController.swift

```diff
@@ -33,20 +35,34 @@ class RemoteTabsTableViewController: UITableViewController,
     private var isShowingEmptyView: Bool { state.showingEmptyState != nil }
     private let emptyView: RemoteTabsEmptyView = .build()
 
+    private var closeTabRemoteDeviceId: String?
+    private var closeTab: RemoteTab?
+    private var tabCommandsFlushTimer: Timer?
+    private let tabCommandsFlushDelay = 6.0
+
     private lazy var longPressRecognizer: UILongPressGestureRecognizer = {
         return UILongPressGestureRecognizer(target: self, action: #selector(longPress))
     }()
 
+    // Checks whether we have allowed the user to close tabs on other
+    // devices
+    private var closeRemoteTabsEnabled: Bool {
+        return featureFlags.isFeatureEnabled(.closeRemoteTabs, checking: .buildAndUser)
```

Comment:
See comment below, we can probably just use https://github.com/mozilla-mobile/firefox-ios/blob/7e5688231e532c568a3589625f6b4695ba160c91/firefox-ios/Client/Nimbus/NimbusFeatureFlagLayer.swift#L314